chapters/methodology/theory.tex

```diff
@@ -144,37 +169,43 @@ \subsubsection{CADIS-$\Omega$}
 particle weights according to their importance. However, CADIS-$\Omega$ will
 use the
 $\Omega$-adjoint scalar flux rather than the standard adjoint scalar flux
-to generate the biased source distribution and the particle weights.
+to generate the biased source distribution, weight windows, and particle birth weights.
 The consistency between the biased source and particle weights is maintained
-when using $\phi_{\Omega}^{\dagger}$ rather than $\phi^{\dagger}$. The adjusted
+when using $\phi_{\Omega}^{\dagger}$ rather than $\phi^{\dagger}$. 
+% I don't understand the above sentence. 
+The adjusted
 formulation of CADIS using the $\Omega$ fluxes is given by Eqs.
-\ref{eq:CADISomegamethod}.
+\eqref{eq:CADISomegamethod}.
+% note: always use eqref--will format equation references more clearly.
```

Comment:
yes. using that is standard practice. The bracketing visually indicates your referring to an equation. You don't have to, but it is what readers in our field are used to.